apps/journalfoer-soeknad/src/main/kotlin/JournalfoerBarnepensjonSoeknadForPesys.kt

```diff
@@ -15,6 +15,9 @@ import org.slf4j.LoggerFactory
 import java.time.Clock
 import java.time.OffsetDateTime
 
+/**
+ * TODO: Denne riveren kan skrotes når fordeleren er ute av verden og vi ikke lenger skal sende bp til pesys
```

Comment:
som nå?

Reply:
Siden fordeleren bare er "skrudd av" med feature toggle, ikke skrotet, lar jeg det ligge enn så lenge. Kommer en mye større opprydning her etter hvert for å samle rivers.